tools/sync-diff-inspector.md

```diff
@@ -18,7 +18,7 @@ sync-diff-inspector 是一个用于校验 MySQL／TiDB 中两份数据是否一
 
 GitHub 地址：[sync-diff-inspector](https://github.com/pingcap/tidb-tools/tree/master/sync_diff_inspector)
 
-下载地址：[sync-diff-inspector-linux-amd64.tar.gz](https://download.pingcap.org/sync-diff-inspector-linux-amd64.tar.gz)
+下载地址：[tidb-latest-linux-amd64](https://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz)
```

Comment:
You also need to update `[tidb-latest-linux-amd64]`.

Reply:
forget update, done